src/Solver.h

```diff
@@ -126,9 +126,9 @@ class Solver {
   /** The total number of Tracks */
   int _tot_num_tracks;
 
-  /** An integer array with the Track uid separating the azimuthal and periodic
-   * halfspaces */
-  int* _num_tracks_by_halfspace;
+  /** An integer array with the Track uid separating the parallel track
+   *  groups */
+  int* _num_tracks_by_parallel_group;
```

Comment:
Can someone explain to me why we are using a pointer to an ``int`` rather than just an ``int``?